### DISCUSSION TITLE
Issue with Ignition and VS Code's Remote - SSH Extension

### DISCUSSION THREAD

**Discussion by HeadStudios** [spatie/laravel-ignition#161] [Q&A]
When using the Remote - SSH extension in Visual Studio Code to work on a Laravel project hosted on a remote server, the Ignition error pages generate file links that try to open files on the local machine, rather than within the remote VS Code session.

**Steps to reproduce**:
1. Set up a Laravel project on a remote server.
2. Use VS Code's Remote - SSH extension to work on the project.
3. Trigger an error in the Laravel application to view it in Ignition.
4. Click on a file link in the Ignition error page.

Video for context:
https://rrdevours.monster/storage/support/issue_vscodeditor.mp4

**Expected behavior**:
The file should open in the remote VS Code session.

**Actual behavior**:
An error message appears indicating that the file path does not exist on the local machine.

**Environment**:
- Laravel version: [your Laravel version]
- Ignition version: [your Ignition version]
- VS Code version: [your VS Code version]
- Remote - SSH extension version: [extension version]

**Comment by renarena** [comment]
I have the same problem

**Reply by parallels999** [reply]
Did you try `phpstorm-remote`?
https://github.com/spatie/ignition/blob/20eab2791feb492e5afe0002cd093155df80fcaa/src/Config/IgnitionConfig.php#L178-L181
